src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days <= 2) {
+    return 40 * days;
+  }
+
+  if (days > 2 && days <= 6) {
+    return 40 * days - 20;
+  }
+
+  if (days >= 7) {
+    return 40 * days - 50;
+  }
```

Comment:
The function name 'calculateRentalCost' is descriptive, which is good. However, the implementation uses numeric literals for the daily cost, discount amounts, and the day thresholds for discounts. Consider defining constants for these values to make the code more readable and maintainable.